src/window/session_window.rs

```diff
@@ -35,7 +35,7 @@ impl WindowBuilder for SessionWindow {
 add_pymethods!(
     SessionWindow,
     parent: WindowConfig,
-    py_args: (gap),
+    signature: (gap=Duration::zero()),
```

Comment:
Since gap is mandatory, we should not add a default value in the signature here:
`signature: (gap)`